hilti/toolchain/src/compiler/codegen/operators.cc

```diff
@@ -815,6 +815,10 @@ struct Visitor : hilti::visitor::PreOrder {
         result = fmt("::hilti::rt::string::startsWith(%s, %s)", self, args[0]);
     }
 
+    void operator()(operator_::string::LowerCase* n) final { result = fmt("::hilti::rt::string::lower(%s)", op0(n)); }
```

Comment:
Tagentially related tech debt: would you mind adding a patch which changes the runtime impl of both `lower` and `upper` so instead of taking `const std::string&` they take `std::string_view`?

This should have no effect for user-provided values which are always owning strings, but could help should we use these functions with internal strings which are something non-owning.

Reply:
scooped this one up into 7b748f914ef8b853dfc83e8ae830d030ad34f577